draft-ietf-httpbis-http2bis.xml

```diff
@@ -3717,49 +3717,96 @@
-          the "reserved (remote)" state.  An excessive number of server push streams can be treated as
-          a <xref target="StreamErrorHandler">stream error</xref> of type
-          <xref target="ENHANCE_YOUR_CALM" format="none">ENHANCE_YOUR_CALM</xref>.
+          The number of <xref target="PUSH_PROMISE" format="none">PUSH_PROMISE</xref> frames is not
+          constrained in the same fashion.  A client that accepts server push SHOULD limit the
+          number of streams it allows to be in the "reserved (remote)" state.  An excessive number
+          of server push streams can be treated as a <xref target="StreamErrorHandler">stream
+          error</xref> of type <xref target="ENHANCE_YOUR_CALM"
+          format="none">ENHANCE_YOUR_CALM</xref>.
         </t>
         <t>
-          Processing capacity cannot be guarded as effectively as state capacity.
-        </t>
-        <t>
-          The <xref target="SETTINGS" format="none">SETTINGS</xref> frame can be abused to cause a peer to expend additional
-          processing time. This might be done by pointlessly changing SETTINGS parameters, setting
-          multiple undefined parameters, or changing the same setting multiple times in the same
-          frame.  <xref target="WINDOW_UPDATE" format="none">WINDOW_UPDATE</xref> or <xref target="PRIORITY" format="none">PRIORITY</xref> frames can be abused to
-          cause an unnecessary waste of resources.
-        </t>
-        <t>
-          Large numbers of small or empty frames can be abused to cause a peer to expend time
-          processing frame headers.  Note, however, that some uses are entirely legitimate, such as
-          the sending of an empty <xref target="DATA" format="none">DATA</xref> or <xref target="CONTINUATION" format="none">CONTINUATION</xref> frame at the
-          end of a stream.
-        </t>
-        <t>
-          Field section compression also offers some opportunities to waste processing resources; see
-          <xref target="COMPRESSION" section="7"/> for more details on potential abuses.
-        </t>
-        <t>
-          Limits in <xref target="SETTINGS" format="none">SETTINGS</xref> parameters cannot be reduced instantaneously, which
-          leaves an endpoint exposed to behavior from a peer that could exceed the new limits. In
-          particular, immediately after establishing a connection, limits set by a server are not
-          known to clients and could be exceeded without being an obvious protocol violation.
+          A number of HTTP/2 implementations were found to be vulnerable to denial of service <xref
+          target="NFLX-2019-002"/>.  The following lists known ways that implementations might be
+          subject to denial of service attack:
         </t>
+
+        <ul>
+          <li>
+            <t>
+              Inefficient tracking of outstanding frames can lead to overload if an adversary can
```

Comment:
```suggestion
              Inefficient tracking of outstanding outbound frames can lead to overload if an adversary can
```

Bit of a nit here but it might be helpful to be as explicit as possible.